src/Config/RuntimeConfigPath.cs

```diff
@@ -233,7 +233,12 @@ private static string GetOverriddenName(string fileName)
             return $"{fileName}.overrides{CONFIG_EXTENSION}";
         }
 
-        private static bool DoesFileExistInCurrentDirectory(string fileName)
+        public static string GetMergedFileNameForEnvironment(string fileName, string environmentValue)
+        {
+            return $"{fileName}{environmentValue}.merged{CONFIG_EXTENSION}";
```

Comment:
Does there need to be a period between `{fileName}{environmentValue}` ? This would be a good opportunity to described merged file properties (name) in the pr description

Reply:
updated